ibm/IBM InfoSphere Big Match for Hadoop v11.5.0.2_odpi_v1.txt

```diff
@@ -0,0 +1,39 @@
+a. Product name / version
+IBM InfoSphere Big Match for Hadoop v11.5.0.2
+http://www-03.ibm.com/software/products/en/infosphere-big-match-for-hadoop
+
+b. Primary contact for questions on this questionnaire
+
+Pankuj Chachra pankuj@us.ibm.com
+
+c. Which technologies governed by the ODPi Runtime Specification does this application interoperate with?
+
+HDFS, HBase, Oozie, MapReduce, Spark
```

Comment:
HBase, Oozie, and Spark are not governed by the ODPi Runtime Specification.  Just HDFS, YARN, and MapReduce are.